.php_cs

```diff
@@ -22,7 +22,6 @@ return Symfony\CS\Config\Config::create()
     ->finder(
         Symfony\CS\Finder\DefaultFinder::create()
             ->exclude('Symfony/CS/Tests/Fixtures')
-            ->notName('ShortTagFixerTest.php')
```

Comment:
Are there other reasons (than the bug we are talking about) why we should exclude this file?

fabbot.io is failing because it does not use the current (fixed) version.